lr-cli/cortx_setup/commands/cluster/reset.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+from cortx_setup.commands.command import Command
+from cortx_setup.commands.common_utils import (
+    get_reset_states
+)
+from provisioner.config import ALL_MINIONS
+from provisioner.commands import (
+    deploy
+)
+from provisioner.salt import (
+    StatesApplier,
+    cmd_run
+)
+class NodeResetCluster(Command):
+    _args = {
+        'type': {
+            'type': str,
+            'optional': True,
+            'help': 'reset all/data'
+        }
+    }
+
+    # deploy the specific component states wrt to stages like reset,cleanup
+    def _deploy(self, components: dict, stage: list = None):
+        for component in components:
+            states = components[component]
+            for state in states:
+                try:
+                    self.logger.debug(
+                        f"Running {state} for ALL_MINIONS"
+                    )
+                    deploy.Deploy()._apply_state(
+                        f'components.{state}', targets= ALL_MINIONS, stages=stage
+                    )
+                except Exception as ex:
+                    raise ex
+
+    def cluster_stop(self):
+        res = cmd_run('pcs cluster stop --all', targets=ALL_MINIONS)
```

Comment:
Do `cortx cluster stop --all` and it has to be run on any one minion...
preferably local minion

Reply:
done